hangouts.cpp

```diff
@@ -95,17 +95,19 @@ SimpleHttps	*https = NULL;
 			https  = new SimpleHttps(hostAndPort);
 		}
 
-		int errorCode;
-		if ((errorCode = https->sendRequest("POST",
+		int resCode = https->sendRequest("POST",
 						    m_url,
 						    headers,
-						    payload.str())) != 200 &&
-		    errorCode == 202)
-		{
+						    payload.str());
+
+		std::string strResCode = to_string(resCode);
+                if(strResCode[0] != '2')
```

Comment:
Why returned code is being converted to string? Can we not check range returned code  instead of single digit ?

Reply:
we are unsure of range returned by the https library, generally it starts with 2 for successful cases.